code-coverage-jacoco/pom.xml

```diff
@@ -110,7 +110,7 @@
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
-                <version>0.7.5.201505241946</version>
+                <version>0.8.2</version>
```

Comment:
I am using `0.8.5` here, IMO this should be merged to master since using `0.7.5.201505241946` results in a broken build.